spec/checkout_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'checkout'
+#require 'menu'
+
+describe Checkout do
+   
+  describe "test view_basket" do
+    context "with 1 item in the basket" do
+      before do
+        # instantiate checkout with fake variable to not use the instance variables of the Menu class
+        # does it need to be a double as it can be passed directly? no need to mock a behaviour
```

Comment:
I personally think it doesn't need to be a double because doing it this way still allows the test to be independent